cybered/hashing/templates/hashing/conclusion.html

```diff
@@ -0,0 +1,38 @@
+<!doctype HTML>
+<html>
+{% include "includes/head.html" %}
+
+<body>
+    {% include "includes/header.html" %}
+    <div class="container">
+        <h3>Conclusion</h3>
+
+        <p>
+            You've completed the CyberEd module on hashing! We started by motivating the need for message
+            digests with a story about Alice, Bob, and Chuck, and then learned about what a message digest is,
+            how a hash function makes one, and what makes a hash function good.
+        </p>
+        <p>
+            After going over the basics, you used a tool to generate message digests for any messages you wanted.
+            If you tested a bunch of different messages, you saw how each hash algorithm generated a digest that
+            was always the same size, and you also saw that even if your message was almost identical to a different
+            message, the digest was completely different.
+        </p>
+        <p>
+            Finally, you saw how keyed hashing could be used to add a password to your message digest, making
+            it incredibly difficult for someone to fake, even if they had the message you sent and the digest
+            you sent with it.
+        </p>
+        <p>
+            Go to the next page to play with hash functions and keyed hashes some more, or head back to the CyberEd
+            homepage to pick a new topic!
+        </p>
+    </div>
+    <div class="container mt-3">
+        <a href="{% url 'hashing:keyed_examples_results' %}" class="btn btn-secondary">Prev</a>
```

Comment:
Add a third link (with the outlined class) to the modules selection? I know that there's one in the footer, but I missed it the first time, so others might as well?

Reply:
Not everyone is going to have a giant screen with the footer miles below the text XD Maybe we should just put a pointer to it at the beginning of the module or something?